src/controllers/MixcloudController.js

```diff
@@ -1,16 +1,37 @@
-controller = new BasicController({
-  supports: {
-    playpause: true,
-    previous: true,
-    next: true
+var isBeta = !!document.querySelector('.mz-player-control');
```

Comment:
Will this ever be not true at the page load, but become true the second time test() is called?  Since it's cached here, it won't test it each time it tries, it will just store whatever the result is the first time this script is loaded